cluster-autoscaler/simulator/clustersnapshot/clustersnapshot.go

```diff
@@ -30,6 +31,19 @@ import (
 type ClusterSnapshot interface {
 	ClusterSnapshotStore
 
+	// AddNodeInfo adds the given NodeInfo to the snapshot without checking scheduler predicates. The Node and the Pods are added,
```

Comment:
why are these additions here, they're already in master branch?

https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/simulator/clustersnapshot/clustersnapshot.go#L71-L82

Reply:
lol nevermind I didn't read far enough :) I see they have moved